pkg/reconciler/verify.go

```diff
@@ -28,18 +29,20 @@ func sFnVerify(_ context.Context, _ *fsm, s *systemState) (stateFn, *ctrl.Result
 
 		for _, cond := range deployment.Status.Conditions {
 			if cond.Type == appsv1.DeploymentAvailable && cond.Status == v1.ConditionTrue {
+				r.log.Debugf("successfully verified keda deployment %s/%s", obj.GetNamespace(), obj.GetName())
 				count++
 			}
 		}
 	}
 
-	if count != 2 {
+	if count != 3 {
+		r.log.Debugf("%d deployments in ready state found ( 3 are expected ) ", count)
 		s.instance.UpdateStateProcessing(
 			v1alpha1.ConditionTypeInstalled,
 			v1alpha1.ConditionReasonVerification,
 			"verification in progress",
 		)
-		return stopWithNoRequeue()
+		return stopWithRequeueAfter(time.Second * 10)
```

Comment:
Got a question here, what is the reason for having this 10sec requeue?

Reply:
the need fot that came up with keda 2.10 where we have 3 workloads (incl webhook wich generated certificate ) and therefore the ready condition is not satisfied in the first cycle and and the check had to be requeued.. (otherwise it was stuck in "Progressing" state.

Reply:
Having discussed it, we decided to include it in separate PR
https://github.com/kyma-project/keda-manager/issues/134